app/models/attachment_visibility.rb

```diff
@@ -43,7 +43,7 @@ def visible?
   def unpublished_edition
     attachable_ids = edition_ids + consultation_ids
     if (unpublishing = Unpublishing.where(edition_id: attachable_ids).first)
-      Edition.unscoped.find(unpublishing.edition_id)
+      Edition.find_by(id: unpublishing.edition_id)
```

Comment:
I think this change means that `#unpublished_edition` will now return `nil` instead of raising an `ActiveRecord::RecordNotFound` exception in the case where the `unpublishing.edition_id` can't be found. Is that OK?

Reply:
@chrisroos:

Yes, that's OK. Now the condition in [this line of `BaseAttachmentsController#fail`][1] will not be met and execution will fall through to [this line][2] thus still resulting in a `404 Not Found`. I'll add something to this effect to the relevant commit note.

[1]: https://github.com/alphagov/whitehall/blob/d509099988dd800f37f9e817eabe461e743a6ccc/app/controllers/base_attachments_controller.rb#L9
[2]: https://github.com/alphagov/whitehall/blob/d509099988dd800f37f9e817eabe461e743a6ccc/app/controllers/base_attachments_controller.rb#L19